### DISCUSSION TITLE
Question regarding recognition

### DISCUSSION THREAD

**Discussion by WouterDM** [jomjol/AI-on-the-edge-device#1768] [Q&A]
Hi all,

Yesterday I tried to install an ai on the edge device for my water counter. For some reason the recognition counts every digit 2 times, giving a raw value that is way too big. It's the first time I encounter this problem and I don't know the reason/solution.
Can someone help me out?

Kind regards,

![Watercounter](https://user-images.githubusercontent.com/45952712/210729693-eb4479f8-e367-4bd9-b4b0-9c136099843f.JPG)

Wouter

**Comment by jomjol** [comment]
Can you post your `config.ini`

**Reply by WouterDM** [reply]
[config.txt](https://github.com/jomjol/AI-on-the-edge-device/files/10351268/config.txt)

**Comment by jomjol** [comment]
You have configured 16 ROIs and that is exactly what you see:

```
[Digits]
Model = /config/dig-cont_0600_s3.tflite
CNNGoodThreshold = 0.5
;LogImageLocation = /log/digit
;LogfileRetentionInDays = 3
main.dig1 205 175 24 43 false
main.dig2 235 175 24 43 false
main.dig3 265 175 24 43 false
main.dig4 295 175 24 43 0
main.dig5 325 175 24 43 0
main.dig6 352 175 24 43 0
main.dig7 382 175 24 43 0
main.dig8 412 175 24 43 0
main.dig1 205 175 24 43 false
main.dig2 235 175 24 43 false
main.dig3 265 175 24 43 false
main.dig4 295 175 24 43 0
main.dig5 325 175 24 43 0
main.dig6 352 175 24 43 0
main.dig7 382 175 24 43 0
main.dig8 412 175 24 43 0
```

Please remove  the second half (e.g. manually) and your problem should be solved.

**Reply by WouterDM** [reply]
That's odd, I guess something went wrong while configuring the ROI's. Thanks a lot for the solution, I will try to fix it when I will be back home.
Keep up the good work!

Kind regards,

Wouter
